source/Weapon.h

```diff
@@ -131,25 +135,13 @@ class Weapon : Entity {
 		m_firing = firing;
 	}
 
-	void setProjectiles(Array<Projectile>* projectileArray) { m_projectiles = projectileArray; };
-	bool scoped() { return m_scoped;  }
-	bool firing() { return m_firing; }
-
-protected:
-	Weapon(shared_ptr<WeaponConfig> config, shared_ptr<Scene>& scene, shared_ptr<Camera>& cam, Array<Projectile>* projectiles) : 
-		m_config(config), m_scene(scene), m_camera(cam), m_projectiles(projectiles) {};
-
-	shared_ptr<ArticulatedModel>    m_viewModel;						///< Model for the weapon
-	shared_ptr<ArticulatedModel>    m_bulletModel;						///< Model for the "bullet"
-	shared_ptr<Sound>               m_fireSound;						///< Sound for weapon firing
-	shared_ptr<AudioChannel>		m_fireAudio;						///< Audio channel for fire sound
+	//void setProjectiles(Array<Projectile>* projectileArray) { m_projectiles = projectileArray; };
```

Comment:
Should this commented out function stay?

Reply:
No it should be removed, will remove in a clean-up pass.